src/std/text.ab

```diff
@@ -32,21 +32,36 @@ pub fun replace_one(source, search, replace) {
     return result
 }
 
-/// Replaces all occurrences of a regex pattern in the content with the provided replace text.
+fun is_gnu_sed(): Bool {
+    trust $ re='\bCopyright\b.+\bFree Software Foundation\b'; [[ \$(sed --version 2>/dev/null) =~ \$re ]] $
+    // We currently ignore the possibility that this can be alpine (busybox) sed
+    // which contains "This is not GNU sed version"
```

Comment:
This sentence implies that alpine, the busybox sed is a GNU sed. Maybe a sentence like:
```suggestion
    // We can't match agains a word "GNU" because
    // alpine's busybox sed returns "This is not GNU sed version"
```